src/daemon/config.rs

```diff
@@ -761,7 +773,7 @@ impl Config {
         }
 
         if let Some(threshold) = self.suspend_child_after_inactive_hours {
-            if threshold < 1 {
+            if threshold < CA_SUSPEND_MIN_HOURS {
                 return Err(ConfigError::other(
                     "suspend_child_after_inactive_hours must be 1 or higher (or not set at all)",
```

Comment:
The constant is set to 2 but the text says 1.

Reply:
fixed